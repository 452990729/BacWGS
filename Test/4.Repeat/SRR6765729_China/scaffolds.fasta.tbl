==================================================
file name: scaffolds.fasta          
sequences:          2517
total length:    1897249 bp  (1897249 bp excl N/X-runs)
GC level:         52.69 %
bases masked:      12564 bp ( 0.66 %)
==================================================
               number of      length   percentage
               elements*    occupied  of sequence
--------------------------------------------------
SINEs:                0            0 bp    0.00 %
      ALUs            0            0 bp    0.00 %
      MIRs            0            0 bp    0.00 %

LINEs:                0            0 bp    0.00 %
      LINE1           0            0 bp    0.00 %
      LINE2           0            0 bp    0.00 %
      L3/CR1          0            0 bp    0.00 %

LTR elements:         0            0 bp    0.00 %
      ERVL            0            0 bp    0.00 %
      ERVL-MaLRs      0            0 bp    0.00 %
      ERV_classI      0            0 bp    0.00 %
      ERV_classII     0            0 bp    0.00 %

DNA elements:         0            0 bp    0.00 %
     hAT-Charlie      0            0 bp    0.00 %
     TcMar-Tigger     0            0 bp    0.00 %

Unclassified:       186        14184 bp    0.75 %

Total interspersed repeats:    14184 bp    0.75 %


Small RNA:            0            0 bp    0.00 %

Satellites:           0            0 bp    0.00 %
Simple repeats:       0            0 bp    0.00 %
Low complexity:       0            0 bp    0.00 %
==================================================

* most repeats fragmented by insertions or deletions
  have been counted as one element
                                                      

The query species was assumed to be homo          
RepeatMasker Combined Database: Dfam_3.0
                                         
run with blastp version 3.0PE-AB [2009-10-30] [linux26-x64-I32LPF64 2009-11-17T18:52:53]
The query was compared to unclassified sequences in ".../RepBase.fasta"

