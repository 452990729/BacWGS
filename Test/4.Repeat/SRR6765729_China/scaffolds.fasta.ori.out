   242 19.69 2.20 10.71 NODE_108_length_2011_cov_2.476483      1734      1824    (187) +      Mariner-N1B_LMi     Unspecified     186     269   (657)    
   307 26.59 0.00 1.27 NODE_108_length_2011_cov_2.476483      1738      1817    (194) +           tRNAThr_CB     Unspecified       6      84     (0)    
   235 16.14 3.08 4.69 NODE_108_length_2011_cov_2.476483      1845      1909    (102) +           SINE2-2_SP     Unspecified      10      73   (178)    
   247 22.23 0.00 1.85 NODE_108_length_2011_cov_2.476483      1957      2011      (0) +           SINE2-1_SP     Unspecified       6      59   (216)    
   312 17.54 0.00 0.00 NODE_1120_length_639_cov_2.119863       347       403    (236) C         tRNA-Lys-AAA     Unspecified    (12)      64       8    
   240 18.46 10.77 0.00 NODE_112_length_1983_cov_1.884336      1833      1897     (86) C           DNA-7-5_NV     Unspecified  (2951)     748     677    
   232 11.75 7.55 3.64 NODE_113_length_1979_cov_2.241164        20        72   (1907) +           P-1_Dpulex     Unspecified    4330    4384   (323)    
   246 16.28 2.33 0.00 NODE_1141_length_628_cov_1.968586       475       517    (111) C         EnSpm-8_CCri     Unspecified   (765)   10924   10881    
   225 13.04 15.38 5.63 NODE_1183_length_605_cov_1.794545        34        98    (507) C         Copia-3_CP-I     Unspecified  (3486)     887     817    
   233 14.29 0.00 2.38 NODE_1183_length_605_cov_1.794545       403       445    (160) +          ATCopia34_I     Unspecified     658     699  (3561)    
   253 12.97 1.79 3.64 NODE_119_length_1928_cov_2.527496      1854      1909     (19) C    piggyBacX-20_CCri     Unspecified  (3736)    1536    1482    
   243 25.93 0.00 0.00 NODE_11_length_3713_cov_2.322581        91       144   (3569) +       Copia-27_PIT-I     Unspecified    2733    2786  (2231)    
   243 17.78 0.00 0.00 NODE_120_length_1924_cov_2.046014      1538      1582    (342) C       Helitron-1_Crp     Unspecified   (959)    4237    4193    
   225 19.55 8.08 7.00 NODE_1273_length_561_cov_2.090909       240       338    (223) +              P-30_HM     Unspecified    2216    2315  (3958)    
   234 11.25 0.00 11.11 NODE_127_length_1909_cov_1.817152      1120      1169    (740) +       Gypsy-28_PSo-I     Unspecified    2539    2583  (3870)    
   250 19.02 6.82 4.44 NODE_1281_length_558_cov_1.737575       139       226    (332) +           SINE2-2_SP     Unspecified       6      95   (156)    
   266 19.70 0.00 3.28 NODE_1281_length_558_cov_1.737575       143       205    (353) +           SINE2-1_VC     Unspecified       6      66   (602)    
   230 17.01 6.25 2.00 NODE_1288_length_556_cov_1.782435        77       124    (432) C        Copia-7_PGr-I     Unspecified  (3759)    1203    1154    
   229 8.85 0.00 5.88 NODE_1293_length_554_cov_1.819639       499       534     (20) C       Gypsy-12_DRh-I     Unspecified  (2996)    1897    1864    
   230 21.84 6.45 6.45 NODE_12_length_3623_cov_2.401626      2684      2776    (847) +        Copia-73_ZM-I     Unspecified    3686    3778  (2352)    
   231 18.02 1.92 3.92 NODE_1303_length_550_cov_1.579798       438       489     (61) C           hAT-42_Lch     Unspecified  (1459)    2016    1966    
   234 20.03 0.00 4.00 NODE_1333_length_540_cov_1.853608       258       309    (231) C            Loa-1_SIn     Unspecified   (816)    4310    4261    
   256 23.73 0.00 1.69 NODE_1358_length_527_cov_1.870763       289       348    (179) +          tRNA-Asn_OS     Unspecified       8      66    (14)    
   239 25.00 0.00 0.00 NODE_1358_length_527_cov_1.870763       385       440     (87) +            ManaSINE1     Unspecified      15      70   (261)    
   230 14.30 0.00 2.86 NODE_13_length_3473_cov_1.984201       110       145   (3328) C          BEL-74_CQ-I     Unspecified  (5823)     346     312    
   228 12.50 25.00 0.00 NODE_13_length_3473_cov_1.984201      3354      3409     (64) C         BEL-22_DTa-I     Unspecified  (3264)    4849    4780    
   242 21.05 1.72 1.72 NODE_1418_length_501_cov_1.813901        19        76    (425) C    Harbinger-46_CCri     Unspecified  (3231)    3410    3353    
   225 18.75 2.08 0.00 NODE_1462_length_485_cov_1.983721       150       197    (288) C        Copia-84_SB-I     Unspecified  (1712)    2805    2757    
   227 15.07 9.26 1.72 NODE_1464_length_483_cov_2.740654       418       471     (12) C         BEL-626_AA-I     Unspecified   (673)    6236    6179    
   239 24.57 0.00 1.75 NODE_1473_length_481_cov_2.262911       414       471     (10) +       Gypsy-15_SCH-I     Unspecified    4607    4663  (6626)    
   236 30.10 3.19 1.04 NODE_152_length_1800_cov_2.491117       396       489   (1311) C           RTEX-3_SSa     Unspecified   (398)    6800    6705    
   392 23.17 0.00 0.00 NODE_154_length_1792_cov_2.507772       853       934    (858) C              ALPINE2     Unspecified    (56)      88       7    
   245 20.70 0.00 1.72 NODE_154_length_1792_cov_2.507772       955      1013    (779) C         tRNA-Lys-AAA     Unspecified    (11)      65       8    
   239 17.57 0.00 3.51 NODE_154_length_1792_cov_2.507772       962      1020    (772) C           SINE2-7_SP     Unspecified   (149)      62       6    
   227 7.46 4.35 2.86 NODE_1572_length_448_cov_1.974555       201       269    (179) C        TguERVL1a_LTR     Unspecified   (421)     256     187    
   228 11.12 0.00 1.85 NODE_1572_length_448_cov_1.974555       215       269    (179) C     Harbinger-8_CCri     Unspecified  (2830)    2574    2521    
   229 7.92 0.00 5.26 NODE_1591_length_443_cov_2.698454       156       195    (248) C        Copia-6_ALY-I     Unspecified  (1436)    2620    2583    
   249 15.87 14.29 0.00 NODE_1591_length_443_cov_2.698454       243       305    (138) +            RTE-7_LCh     Unspecified    2532    2603  (1403)    
   236 17.43 8.96 5.80 NODE_15_length_3217_cov_2.048071      2524      2590    (627) +       Gypsy-12_Sit-I     Unspecified    2605    2673  (1729)    
   259 21.19 0.00 5.63 NODE_160_length_1755_cov_2.188824       793       867    (888) C       CryptonS-23_PI     Unspecified   (242)    2659    2589    
   322 23.61 0.00 0.00 NODE_160_length_1755_cov_2.188824       801       872    (883) +         tRNA-Tyr-TAC     Unspecified       5      76     (0)    
   226 18.75 1.56 8.33 NODE_160_length_1755_cov_2.188824      1160      1223    (532) +        Gypsy-2_CSa-I     Unspecified    2941    3000  (2721)    
   269 12.50 5.36 0.00 NODE_161_length_1753_cov_1.992933       712       767    (986) C        Gypsy-91_ZM-I     Unspecified  (4549)      73      15    
   232 11.65 5.08 12.73 NODE_163_length_1747_cov_2.094563      1257      1315    (432) +           TE-10_LSal     Unspecified     315     369   (110)    
   287 25.37 2.99 0.00 NODE_1644_length_424_cov_1.869919        26        92    (332) +        tRNAIleATT_CB     Unspecified       3      71    (10)    
   230 20.01 0.00 2.22 NODE_1670_length_416_cov_2.072022       363       408      (8) C       Gypsy-7_OS-LTR     Unspecified  (2551)    1735    1691    
   242 10.87 6.52 0.00 NODE_1688_length_410_cov_1.766197       226       271    (139) C            L1-30_AAe     Unspecified  (2173)    3364    3316    
   233 6.67 0.00 0.00 NODE_169_length_1716_cov_2.338952       644       673   (1043) +        FEILAI-1B_AAe     Unspecified      50      79   (210)    
   240 32.88 0.00 1.37 NODE_169_length_1716_cov_2.338952       707       780    (936) +         tRNA-Arg2_OS     Unspecified       1      73     (0)    
   231 24.19 0.00 0.00 NODE_169_length_1716_cov_2.338952       811       872    (844) +     BivaV-SINE3_CoAn     Unspecified       3      64   (206)    
   236 12.20 12.20 0.00 NODE_172_length_1700_cov_1.688146      1427      1467    (233) +        Copia-75_AA-I     Unspecified    1171    1216  (3142)    
   231 17.39 2.17 0.00 NODE_1735_length_389_cov_1.886228       307       352     (37) +        Copia-92_SB-I     Unspecified    4037    4083   (308)    
   491 33.04 2.33 2.33 NODE_1753_length_383_cov_1.554878        14       357     (26) C        EnSpm-26_CCri     Unspecified  (7126)    2957    2614    
   225 27.40 2.74 0.00 NODE_1778_length_377_cov_2.086957        87       159    (218) +       Gypsy-108_AA-I     Unspecified    1842    1916  (2703)    
   230 10.54 0.00 2.63 NODE_1795_length_373_cov_1.481132       135       173    (200) C              P-22_HM     Unspecified    (35)    3130    3093    
   235 20.03 1.47 4.55 NODE_17_length_3159_cov_2.967461      1264      1331   (1828) +        Gypsy-18_AA-I     Unspecified    2860    2925  (1460)    
   229 20.83 8.33 0.00 NODE_180_length_1672_cov_2.244280       691       738    (934) +          Nimb-31_LMi     Unspecified    4824    4875  (1066)    
   230 6.65 10.42 6.00 NODE_1831_length_362_cov_2.013029       254       301     (61) +                R7Ag2     Unspecified    2761    2810  (3780)    
   248 24.08 1.82 1.82 NODE_18_length_3145_cov_2.211327      1932      1986   (1159) +       Gypsy-160_AA-I     Unspecified    3540    3594  (2493)    
   238 15.92 0.00 2.27 NODE_1981_length_325_cov_1.533333       131       175    (150) +           BEL30-I_DR     Unspecified    1959    2002  (4646)    
   225 32.00 0.00 0.00 NODE_1_length_7618_cov_3.306757      2327      2426   (5192) +         SSU-rRNA_Lma     Unspecified    1163    1262   (875)    
   661 32.22 7.87 3.25 NODE_1_length_7618_cov_3.306757      2366      3013   (4605) +         SSU-rRNA_Tps     Unspecified     981    1657   (147)    
   357 28.43 5.51 11.36 NODE_1_length_7618_cov_3.306757      2755      3135   (4483) +     SSU-rRNA_Giardia     Unspecified    1092    1452    (12)    
   266 27.37 3.53 1.15 NODE_1_length_7618_cov_3.306757      3241      3325   (4293) +           SINE2-2_SP     Unspecified       2      88   (163)    
   296 23.73 0.00 0.00 NODE_1_length_7618_cov_3.306757      3251      3309   (4309) +          tRNA-Asn_OS     Unspecified       8      66    (14)    
   345 24.18 0.00 1.10 NODE_1_length_7618_cov_3.306757      3326      3417   (4201) +              ALPINE1     Unspecified       4      94    (33)    
   252 27.08 4.95 4.95 NODE_1_length_7618_cov_3.306757      3333      3433   (4185) +           SINE2-1_PL     Unspecified      13     113   (135)    
   304 25.28 0.00 1.10 NODE_1_length_7618_cov_3.306757      3917      4008   (3610) +         LSU-rRNA_Hca     Unspecified      35     125  (3530)    
   321 24.14 0.00 0.00 NODE_1_length_7618_cov_3.306757      3929      4015   (3603) +         LSU-rRNA_Ddi     Unspecified      23     109  (3132)    
   315 19.99 4.55 3.37 NODE_1_length_7618_cov_3.306757      4146      4233   (3385) +         LSU-rRNA_Ddi     Unspecified     293     381  (2860)    
   390 23.96 0.00 0.00 NODE_1_length_7618_cov_3.306757      4168      4263   (3355) +         LSU-rRNA_Hca     Unspecified     338     433  (3222)    
   292 24.50 0.99 3.03 NODE_1_length_7618_cov_3.306757      4191      4291   (3327) +         LSU-rRNA_Ldo     Unspecified      11     109   (881)    
   264 29.79 3.23 2.56 NODE_1_length_7618_cov_3.306757      4414      4568   (3050) +     LSU-rRNA_Giardia     Unspecified     518     673  (2017)    
  1043 22.59 3.09 4.38 NODE_1_length_7618_cov_3.306757      4700      5023   (2595) C        Copia-18_BD-I     Unspecified    (64)    4768    4449    
   279 27.42 4.92 14.80 NODE_1_length_7618_cov_3.306757      5345      5588   (2030) +     LSU-rRNA_Giardia     Unspecified    1472    1694   (996)    
   713 25.39 2.73 5.12 NODE_1_length_7618_cov_3.306757      5512      5731   (1887) +     LSU-rRNA_Giardia     Unspecified    1616    1830   (860)    
   475 28.94 6.25 8.72 NODE_1_length_7618_cov_3.306757      5515      5866   (1752) +         LSU-rRNA_Hca     Unspecified    2411    2754   (901)    
   567 27.66 6.42 6.42 NODE_1_length_7618_cov_3.306757      5943      6425   (1193) +         LSU-rRNA_Hsa     Unspecified    4123    4605   (430)    
   598 30.00 4.52 4.52 NODE_1_length_7618_cov_3.306757      5947      6344   (1274) +         LSU-rRNA_Dme     Unspecified    3042    3439   (461)    
   492 31.57 4.32 4.70 NODE_1_length_7618_cov_3.306757      5954      6508   (1110) +         LSU-rRNA_Cel     Unspecified    2699    3251   (258)    
   251 22.24 1.33 4.11 NODE_2011_length_316_cov_1.754789       217       291     (25) +       Mariner3-4_LMi     Unspecified    2787    2859   (172)    
   256 23.84 0.00 3.17 NODE_2011_length_316_cov_1.754789       217       281     (35) +             TE-3_LVa     Unspecified     820     882  (1778)    
   271 21.84 3.33 3.33 NODE_2022_length_313_cov_1.810078         1        90    (223) C                RTAg4     Unspecified  (4877)    2195    2106    
   230 29.89 4.60 0.00 NODE_2076_length_300_cov_1.844898        95       181    (119) C        Gypsy-73_MN-I     Unspecified  (1223)    5516    5426    
   226 10.81 2.70 0.00 NODE_209_length_1569_cov_1.906209      1314      1350    (219) C       Gypsy-258_AA-I     Unspecified  (2219)    3033    2996    
   247 24.34 1.35 5.63 NODE_211_length_1567_cov_1.792328      1026      1099    (468) C          Copia3-I_CR     Unspecified  (1450)    3890    3820    
   257 19.76 3.57 3.57 NODE_2191_length_273_cov_2.110092       160       243     (30) C             Ves2B_ML     Unspecified   (162)      90       7    
   317 18.97 0.00 0.00 NODE_2191_length_273_cov_2.110092       181       238     (35) C         SINE2-2B_SSa     Unspecified   (116)      66       9    
   235 21.61 1.19 11.84 NODE_2191_length_273_cov_2.110092       182       265      (8) C            hAT-51_HM     Unspecified   (210)    3624    3549    
   253 11.43 0.00 0.00 NODE_2225_length_264_cov_2.641148        23        57    (207) C       Gypsy-69_PAb-I     Unspecified  (4037)     913     879    
   226 27.42 3.23 0.00 NODE_225_length_1532_cov_1.800271      1336      1397    (135) +        Zator-1N1_LCh     Unspecified     214     277   (651)    
   241 19.05 0.00 0.00 NODE_227_length_1526_cov_2.095853       438       479   (1047) +        Jockey-2_DGri     Unspecified    3707    3748  (2284)    
   234 24.53 3.77 0.00 NODE_228_length_1525_cov_2.151020      1457      1509     (16) C          EnSpm-N1_OS     Unspecified  (3178)    1078    1024    
   236 20.34 8.47 0.00 NODE_232_length_1513_cov_2.169410       582       640    (873) +         ERV2-3_Pca-I     Unspecified     902     965  (6011)    
   230 14.65 0.00 2.44 NODE_239_length_1501_cov_2.081604       279       320   (1181) +        Copia-1_ALY-I     Unspecified     185     225  (4573)    
   240 21.01 10.26 2.38 NODE_241_length_1497_cov_2.253814      1261      1338    (159) +           Tad1-34_BG     Unspecified    5186    5269   (741)    
   249 20.04 0.00 4.29 NODE_2461_length_214_cov_1.610063         2        74    (140) +               R1_DYa     Unspecified     760     829  (4684)    
   235 17.54 8.06 8.06 NODE_254_length_1461_cov_1.903983       287       348   (1113) +     Helitron-1_PTrit     Unspecified    4112    4173   (257)    
   227 19.25 14.29 1.05 NODE_263_length_1431_cov_2.483285       164       247   (1184) +           SART-5_APi     Unspecified    1902    1996  (2409)    
   233 24.07 7.41 0.00 NODE_264_length_1425_cov_2.226277       824       877    (548) C          EnSpm-11_ZM     Unspecified   (110)    7816    7759    
   229 18.18 0.00 0.00 NODE_265_length_1424_cov_1.938641       155       198   (1226) +             GYZMA1_I     Unspecified    3072    3115  (1695)    
   227 6.06 3.03 0.00 NODE_266_length_1423_cov_2.200292      1124      1156    (267) C           DNA2-25_CB     Unspecified   (574)     412     379    
   248 15.00 0.00 11.11 NODE_275_length_1397_cov_1.932936       150       209   (1188) +      Harbinger-1_MLP     Unspecified    3877    3930  (1193)    
   226 17.65 13.73 0.00 NODE_27_length_2915_cov_2.297902      2205      2255    (660) +          BEL-15_AA-I     Unspecified    1888    1945  (3423)    
   284 26.23 6.86 2.83 NODE_281_length_1385_cov_2.919549       462       563    (822) +            GarnAlu3B     Unspecified       8     113   (147)    
   248 20.00 1.64 1.64 NODE_289_length_1370_cov_2.276806      1219      1279     (91) +     Harbinger-2C_CPB     Unspecified     384     444  (4068)    
   234 14.29 0.00 0.00 NODE_294_length_1363_cov_2.146789      1120      1154    (209) +        Gypsy-13_OD-I     Unspecified    3349    3383  (3810)    
   233 21.95 0.00 0.00 NODE_2_length_7259_cov_3.931011      3607      3647   (3612) C        Gypsy-1_CMe-I     Unspecified  (2155)   10255   10215    
   227 18.18 6.82 0.00 NODE_2_length_7259_cov_3.931011      4449      4492   (2767) C      Mariner-5_PTrit     Unspecified  (2915)     668     622    
   230 24.00 0.00 0.00 NODE_302_length_1342_cov_2.272727       660       709    (633) +           hATm-41_HM     Unspecified     185     234  (2750)    
   242 27.65 0.00 7.50 NODE_317_length_1323_cov_2.053628      1121      1206    (117) +        tRNA-Ser-TCA_     Unspecified       6      85     (0)    
   227 19.60 0.00 11.29 NODE_322_length_1314_cov_1.869738       938      1006    (308) C             SZ-66LTR     Unspecified    (84)    1139    1078    
   226 31.63 0.00 0.00 NODE_326_length_1306_cov_2.392486       497       594    (712) C              TART_DV     Unspecified  (7943)    7156    7059    
   230 26.39 4.17 0.00 NODE_326_length_1306_cov_2.392486       497       568    (738) C        Jockey-2_DGri     Unspecified  (5294)     738     664    
   241 20.97 0.00 0.00 NODE_326_length_1306_cov_2.392486       497       558    (748) C            Ag-Jock-1     Unspecified  (3261)    1464    1403    
   246 9.09 0.00 0.00 NODE_32_length_2820_cov_1.844123      2007      2039    (781) C       Gypsy22-I_Dpse     Unspecified  (5271)    3379    3347    
   312 27.40 0.00 0.00 NODE_37_length_2729_cov_2.062453      1077      1149   (1580) +         tRNA-Arg2_OS     Unspecified       1      73     (0)    
   235 26.39 2.70 2.70 NODE_37_length_2729_cov_2.062453      1084      1157   (1572) C       Gypsy-14_DAn-I     Unspecified  (6026)      79       6    
   255 23.66 0.00 4.17 NODE_383_length_1226_cov_1.794193       821       895    (331) C              SINE_SM     Unspecified    (76)      85      14    
   312 17.24 0.00 0.00 NODE_383_length_1226_cov_1.794193       834       891    (335) C         tRNA-Lys-AAG     Unspecified    (11)      65       8    
   227 13.05 2.13 2.13 NODE_384_length_1225_cov_2.685470      1025      1071    (154) +        Penelope-1_DR     Unspecified     524     570  (3876)    
   233 14.27 8.33 6.56 NODE_386_length_1223_cov_1.875856       157       216   (1007) C       Helitron-3_ALy     Unspecified  (8835)    6161    6101    
   245 13.21 7.55 0.00 NODE_394_length_1212_cov_2.136560       220       272    (940) C          Jockey-4_DK     Unspecified  (1371)    2750    2694    
   227 24.00 0.00 0.00 NODE_398_length_1208_cov_2.224631        21        70   (1138) C       Gypsy-18_ADe-I     Unspecified  (2135)    8707    8658    
   235 17.65 1.92 1.92 NODE_3_length_6917_cov_4.690469      4243      4294   (2623) C         Gypsy-9_RP-I     Unspecified  (1160)    4491    4440    
   238 2.80 0.00 18.92 NODE_40_length_2682_cov_1.928055       986      1029   (1653) +          LINE1-31_ZM     Unspecified    1047    1083  (2599)    
   225 9.28 13.64 2.04 NODE_417_length_1191_cov_2.297535       719       762    (429) C        Gypsy-1_PGr-I     Unspecified  (2056)    5764    5716    
   240 22.45 2.04 0.00 NODE_417_length_1191_cov_2.297535      1027      1075    (116) C     Gypsy-11_CFl-LTR     Unspecified   (909)     306     257    
   230 17.87 1.72 3.51 NODE_425_length_1181_cov_1.955595       415       472    (709) C       Gypsy-15_ATr-I     Unspecified  (2772)   11174   11118    
   239 17.39 2.17 0.00 NODE_453_length_1133_cov_2.418367       773       818    (315) +       Helitron-1_AEc     Unspecified    5637    5683  (2124)    
   230 12.51 0.00 2.50 NODE_48_length_2583_cov_1.855222      1124      1164   (1419) +        Polinton-2_TC     Unspecified   16912   16951    (30)    
   236 26.67 5.33 0.00 NODE_494_length_1086_cov_2.575170        42       116    (970) C         Ogre-PT2_LTR     Unspecified   (680)    2095    2017    
   231 15.64 4.41 5.97 NODE_494_length_1086_cov_2.575170       561       628    (458) +             Tx1-4_DR     Unspecified    3256    3322  (2213)    
   278 17.65 0.00 0.00 NODE_497_length_1082_cov_5.484907       229       279    (803) C        Mariner-1_PSo     Unspecified   (410)    1251    1201    
   230 18.95 5.08 1.64 NODE_497_length_1082_cov_5.484907       232       290    (792) C        Mariner-4_LMi     Unspecified   (429)    1197    1137    
   255 33.28 9.02 2.84 NODE_4_length_6403_cov_2.728261      1966      2231   (4172) +         Gypsy18-I_CR     Unspecified    2633    2914  (3223)    
   465 23.34 7.50 3.61 NODE_4_length_6403_cov_2.728261      1980      2139   (4264) +       Gypsy-32_ADe-I     Unspecified    1900    2065  (9464)    
   460 29.42 1.43 2.90 NODE_4_length_6403_cov_2.728261      1996      2205   (4198) +        MarinerL-1_CR     Unspecified    1399    1605 (14383)    
   471 28.40 3.55 0.00 NODE_4_length_6403_cov_2.728261      2025      2193   (4210) +       Gypsy-32_ADe-I     Unspecified    1891    2065  (9464)    
   235 26.50 6.93 2.86 NODE_52_length_2504_cov_2.024500      1130      1230   (1274) +            hAT-10_XT     Unspecified      59     163 (10981)    
   225 20.00 0.00 0.00 NODE_532_length_1054_cov_1.737738       562       601    (453) C        Copia-56_BD-I     Unspecified   (457)    4757    4718    
   234 18.52 14.81 0.00 NODE_53_length_2480_cov_2.079588      2226      2279    (201) C         BEL-617_AA-I     Unspecified  (2199)    3649    3588    
  1243 29.26 1.36 1.36 NODE_543_length_1040_cov_2.235533       114       553    (487) +        EnSpm-29_CCri     Unspecified    4269    4708  (6265)    
   233 12.83 0.00 2.56 NODE_55_length_2447_cov_2.096572      1464      1503    (944) C           Tad1-18_BG     Unspecified   (319)    5459    5421    
   234 26.03 4.11 0.00 NODE_587_length_996_cov_1.720510       128       200    (796) C         Gypsy-9-I_CR     Unspecified  (1469)   10935   10860    
   241 12.49 4.08 2.00 NODE_587_length_996_cov_1.720510       807       855    (141) C        Gypsy-15_CQ-I     Unspecified  (2932)    2324    2275    
   238 28.17 0.00 0.00 NODE_616_length_972_cov_2.323882        46       116    (856) C      Helitron-N3_CGi     Unspecified  (5118)    2221    2151    
   303 21.05 0.00 0.00 NODE_616_length_972_cov_2.323882        56       112    (860) C         tRNA-Tyr-TAC     Unspecified    (14)      62       6    
   242 15.38 0.00 0.00 NODE_617_length_972_cov_2.279171       413       451    (521) C      Polinton-1N1_DR     Unspecified  (7866)    7603    7565    
   225 25.81 0.00 4.55 NODE_646_length_937_cov_1.667800       818       886     (51) C        MarinerL-1_CR     Unspecified (12675)    3313    3248    
   225 28.39 0.00 2.99 NODE_646_length_937_cov_1.667800       831       899     (38) C        MarinerL-1_CR     Unspecified (14529)    1459    1393    
   234 22.25 0.00 3.70 NODE_646_length_937_cov_1.667800       831       886     (51) C        MarinerL-1_CR     Unspecified (14493)    1495    1442    
   227 22.92 0.00 0.00 NODE_663_length_924_cov_2.016110       358       405    (519) +         Jockey-N4_CQ     Unspecified    1233    1280   (513)    
   235 26.58 1.52 3.08 NODE_66_length_2269_cov_2.175700       554       619   (1650) C         BEL-615_AA-I     Unspecified  (4329)    1854    1790    
   247 26.58 1.27 0.00 NODE_683_length_901_cov_2.290780        99       177    (724) +        Gypsy-6_PIT-I     Unspecified    6217    6296   (663)    
   240 24.62 1.54 0.00 NODE_683_length_901_cov_2.290780       194       258    (643) C         tRNA-Ala-GCA     Unspecified     (7)      68       3    
   250 21.43 1.79 0.00 NODE_683_length_901_cov_2.290780       200       255    (646) +        Gypsy-6_PIT-I     Unspecified    6114    6170   (789)    
   235 15.23 9.38 7.69 NODE_683_length_901_cov_2.290780       301       364    (537) C       tRNA-SerCGA_OS     Unspecified     (7)      75      11    
   230 16.68 2.00 4.08 NODE_692_length_896_cov_2.180737       360       409    (487) C         Gypsy65-ZM_I     Unspecified   (512)    5101    5053    
   242 21.93 0.00 7.27 NODE_69_length_2264_cov_5.064282       252       310   (1954) C        Gypsy-1_CDC-I     Unspecified  (1152)    4213    4159    
   261 24.03 6.67 0.90 NODE_69_length_2264_cov_5.064282      1460      1564    (700) +              CACTA-N     Unspecified     585     695   (292)    
   263 13.46 13.33 0.99 NODE_69_length_2264_cov_5.064282      1476      1565    (699) +           DNA9-20_OS     Unspecified    1533    1633   (212)    
   307 13.54 2.91 13.98 NODE_69_length_2264_cov_5.064282      1490      1592    (672) C           MERMITE18E     Unspecified  (1225)     299     207    
   243 6.78 3.39 0.00 NODE_69_length_2264_cov_5.064282      2204      2262      (2) C       Gypsy-133_GM-I     Unspecified  (9509)    2283    2223    
   241 24.27 0.00 3.03 NODE_6_length_4190_cov_2.120193      1707      1774   (2416) +        Daphne-11_CTe     Unspecified     471     536  (3778)    
   226 29.49 1.28 0.00 NODE_712_length_883_cov_2.496377       722       799     (84) C          BEL-2_LMi-I     Unspecified   (927)    4938    4860    
   273 20.75 5.68 6.90 NODE_712_length_883_cov_2.496377       732       819     (64) +          BEL-2_LMi-I     Unspecified    5223    5309   (556)    
   329 21.62 0.00 1.35 NODE_713_length_883_cov_2.078502       513       587    (296) C         tRNA-Lys-AAA     Unspecified     (0)      76       3    
   234 23.72 3.33 1.64 NODE_724_length_874_cov_1.835165       629       688    (186) +          ERV1-2-I_BT     Unspecified    5068    5128  (1158)    
   250 30.23 3.64 7.56 NODE_740_length_859_cov_2.077114        91       337    (522) C          CRE-15_CCri     Unspecified   (121)    2327    2090    
   226 14.29 2.38 0.00 NODE_754_length_848_cov_2.035309       637       678    (170) +            MuDr-5_HM     Unspecified    7331    7373  (2804)    
   236 29.07 15.12 0.00 NODE_757_length_845_cov_1.855696        73       158    (687) C        Gypsy-30_OD-I     Unspecified  (5464)    1113    1015    
   230 32.75 0.00 5.44 NODE_772_length_832_cov_1.446589       626       780     (52) +        Sola1-2C_CCri     Unspecified    2813    2959  (9425)    
   240 14.29 0.00 0.00 NODE_801_length_810_cov_2.064901       442       476    (334) C       Gypsy-35_AIp-I     Unspecified  (4389)     115      81    
   230 21.15 1.92 0.00 NODE_802_length_810_cov_2.002649       361       412    (398) C            Tad1-1_BG     Unspecified   (286)    6050    5998    
   239 20.52 13.51 1.20 NODE_80_length_2156_cov_2.338886      1181      1254    (902) +        Helitron-1_SP     Unspecified    8027    8109  (1991)    
   237 22.22 0.00 0.00 NODE_832_length_795_cov_2.106757       423       467    (328) C         ERV1-4_TSy-I     Unspecified   (807)    4307    4263    
   230 20.44 0.00 4.08 NODE_852_length_786_cov_2.823529       111       161    (625) C      Gypsy-36_PMar-I     Unspecified   (802)    2440    2392    
   235 15.76 12.26 10.19 NODE_871_length_773_cov_2.551532       498       603    (170) C          EnSpm-1_AIp     Unspecified    (74)    5474    5367    
   229 13.15 0.00 17.02 NODE_88_length_2088_cov_2.203148      1038      1092    (996) C          Gypsy5-VV_I     Unspecified   (132)    4430    4384    
   227 18.75 2.08 0.00 NODE_89_length_2086_cov_1.989660       292       339   (1747) +       Chapaev-1N1_NV     Unspecified     535     583  (2001)    
   242 22.42 0.00 1.72 NODE_913_length_748_cov_2.034632       613       671     (77) +       Copia-39_AIp-I     Unspecified     222     279  (4631)    
   226 15.00 7.50 0.00 NODE_931_length_735_cov_1.875000       630       669     (66) C           DNA5-1_PSt     Unspecified    (44)     377     335    
   225 15.91 4.55 0.00 NODE_956_length_723_cov_1.839820       652       695     (28) +          Gypsy6-I_VC     Unspecified    3533    3578  (1352)    
   256 23.73 1.69 0.00 NODE_960_length_719_cov_3.031627       393       451    (268) +        Gypsy-3_CFl-I     Unspecified    1687    1746  (3706)    
   234 10.26 0.00 2.56 NODE_992_length_703_cov_2.490741       526       565    (138) +          DNA-6-10_NV     Unspecified    1127    1165  (1239)    
